app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -270,6 +285,49 @@ class GithubWebhook extends RequestHandler<Body> {
     await buildBucketClient.batch(BatchRequest(requests: requests));
   }
 
+  Future<bool> _isIgnoredForGold(PullRequestEvent event) async {
+    bool ignored = false;
+    try {
+      final HttpClientRequest request = await skiaClient.getUrl(
+        Uri.parse('https://flutter-gold.skia.org/json/ignores')
+      );
+      final HttpClientResponse response = await request.close();
+      final String rawResponse = await utf8.decodeStream(response);
+      final List<dynamic> ignores = jsonDecode(rawResponse);
+      for (Map<String, dynamic> ignore in ignores) {
+        if (ignore['note'].isNotEmpty &&
+          event.number.toString() == ignore['note'].split('/').last) {
+          ignored = true;
+          break;
+        }
+      }
+    } catch(_) {}
```

Comment:
What are we actually trying to catch here?  We should generally avoid catch blocks that catch all exceptions, and also avoid catch blocks that swallow exceptions.

Reply:
I figured that if there is an error making the request it should just return false. If (let us hope never) Skia Gold is down and the request fails, I wouldn't want it to have a domino effect and break the bot as a result.

Reply:
I think in that case we should just try to catch `SocketException` and maybe `HttpException`, and just print something to the log that we couldn't reach Skia Gold.

We may also want to catch `FormatException` and print out the raw data we got and rethrow in case the response format changes.

Reply:
SGTM! I'll put it together! :)

Reply:
Or maybe just catch `IOException` instead of `Socket/Http` separately.